File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n=nums.length;
        int a[]=new int[2*n];
        int temp=n;
        for(int i=0;i<n;i++){
            a[i]=nums[i];
            a[temp]=nums[i];
            temp++;
        }
        return a;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        res=[]
        c=0
        for i in nums:
            if i==1:
                c+=1
            else:
                res.append(c)
                c=0
        res.append(c)
        return max(res)